pulsar-broker/src/test/java/org/apache/pulsar/broker/service/GracefulExecutorServicesShutdownTest.java

```diff
@@ -152,7 +153,8 @@ public void shouldTerminateWhenFutureIsCancelled() throws InterruptedException,
         // when
         shutdown.shutdown(executorService);
         CompletableFuture<Void> future = shutdown.handle();
-        awaitingTerminationEntered.await();
+        // waiting to start awaitTermination
+        verify(executorService, timeout(100).atLeastOnce()).awaitTermination(anyLong(), any());
```

Comment:
I don't know why this change was made, but isn't `awaitingTerminationEntered` no longer needed?

Reply:
I didn't notice that the problem was already fixed by `awaitingTerminationEntered`. I'll remove this changes.